modules/common/util/string.go

```diff
@@ -25,3 +30,15 @@ func StringInSlice(a string, list []string) bool {
 	}
 	return false
 }
+
+// RandomString - generate a random string of given length
+func RandomString(length int) []byte {
```

Comment:
could also just use https://pkg.go.dev/k8s.io/apimachinery/pkg/util/rand#String ?

Reply:
Ah, that makes sense. Let me try it.